frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ReleasePlanChange.tsx

```diff
@@ -177,24 +146,107 @@ const StartMilestone: FC<{
 
 const AddReleasePlan: FC<{
     change: IChangeRequestAddReleasePlan;
+    currentReleasePlan?: IReleasePlan;
     environmentName: string;
     featureName: string;
     actions?: ReactNode;
-}> = ({ change, environmentName, featureName, actions }) => {
+}> = ({
+    change,
+    currentReleasePlan,
+    environmentName,
+    featureName,
+    actions,
+}) => {
+    const [currentTooltipOpen, setCurrentTooltipOpen] = useState(false);
+    const currentTooltipCloseTimeoutRef = useRef<NodeJS.Timeout>();
+    const openCurrentTooltip = () => {
+        if (currentTooltipCloseTimeoutRef.current) {
+            clearTimeout(currentTooltipCloseTimeoutRef.current);
+        }
+        setCurrentTooltipOpen(true);
+    };
+    const closeCurrentTooltip = () => {
+        currentTooltipCloseTimeoutRef.current = setTimeout(() => {
+            setCurrentTooltipOpen(false);
+        }, 100);
+    };
```

Comment:
Unfortunately the default tooltip behavior did not work correctly with the accordions. If you toggled one of them after the other, it would close the tooltip.

This required some manual plumbing (manually handle the tooltip open state), but the final result seems worth it.